changedetectionio/forms.py

```diff
@@ -342,8 +343,8 @@ def __call__(self, form, field):
                 try:
                     elementpath.select(tree, line.strip(), parser=XPath3Parser)
                 except elementpath.ElementPathError as e:
-                    message = field.gettext('\'%s\' is not a valid XPath expression. (%s)')
-                    raise ValidationError(message % (line, str(e)))
+                    message = _("'{}' is not a valid XPath expression. ({})")
```

Comment:
I tested with wrong xpath syntax and worked. But am I right?
I'm totally unaware of the `field.gettext`. If it is come from wtforms, the gettext is from python builtin lib.